standards_lab/api/views.py

```diff
@@ -140,15 +148,27 @@ def delete(self, *args, **kwargs):
 
 class ProjectDownloadFile(View):
     def get(self, request, *args, **kwargs):
+        type = kwargs["type"]
         project = get_project_config(kwargs["name"])
 
-        # Downloading is limited to files tracked in our project
-        if not kwargs["file_name"] in project.get("schemaFiles", []) + project.get(
-            "dataFiles", []
-        ):
+        # check only allowed types & files tracked in our project
+        if type == "data":
+            if not kwargs["file_name"] in project.get("dataFiles", []):
```

Comment:
```python
if type == "data" and not kwargs["file_name"] in project.get("dataFiles", []): 
``` 
Although all this becomes simpler if you just use "dataFiles" as a directory name. 

```python
if kwargs["file_name"] in project.get(type, []):
```

Reply:
> Can be simplified as ...

Don't think so? If type == "data" and the file "data.json" was requested and "data.json" is in project['datafiles'] - ie, the request is for a data file that does exist - then the else clause several lines would below would trigger and a 404 would be thrown, no?

I've changed comments to try and make clearer what I'm doing.

Reply:
Ah yes my bad, didn't clock the last `else` which would be triggered if you combined those if conditions.